package.json

```diff
@@ -28,48 +28,48 @@
   "author": "Grafana Labs",
   "license": "MIT",
   "devDependencies": {
```

Comment:
The only relevant change is in the package.json, the rest is just formatting by prettier